pkgs/development/python-modules/timezonefinder/default.nix

```diff
@@ -60,12 +62,12 @@ buildPythonPackage rec {
     "tests/utils_test.py"
   ];
 
-  meta = with lib; {
```

Comment:
This is okay though as it's a treewide policy: https://github.com/NixOS/nixpkgs/issues/208242